### DISCUSSION TITLE
关于更新的问题

### DISCUSSION THREAD

**Discussion by Li1177** [ChatGPTNextWeb/NextChat#1057] [Q&A]
很棒的软件按照作者的建议配置在versel上，并且做了域名等等的设置，变成了一个很顺手的工具。
现在想更新了，看到教程说：需要fork的方式，并且删除原来的仓库。
因为纯小白，所以只能猜：意思是不是要删除掉现有versel里的这个项目，让后用fork的方式重新导入到versel里是吗？
那么问题来了，之前的域名相关的设置是不是要重新做了啊？
然后改fork的方式部署到versel上以后，以后更新可以基本上就是自动了是吗？不会再做一些需要重设置
域名的操作了，是吗

**Comment by Yidadaa** [comment]
你的理解是对的。

**Reply by Li1177** [reply]
好吧，回头等个大更新再说
